lib/install/tailwindcss_with_webpacker.rb

```diff
@@ -7,7 +7,6 @@
 
 say "Configuring Tailwind CSS"
 directory Pathname.new(__dir__).join("stylesheets"), Webpacker.config.source_path.join("stylesheets")
-Dir.chdir(WEBPACK_STYLESHEETS_PATH) { run "npx tailwindcss init" }
```

Comment:
Is this not needed anymore? Or what is the connection from this line to the purging config of this PR?

Reply:
Yup not needed anymore, since I moved `tailwind.config.js` to "templates", reason is purge configuration.
It's being copied on the line above(9) together with `application.css`.